### DISCUSSION TITLE
Why Noi Ask is not working with ChatGPT？为什么Noi Ask在ChatGPT里不行呢？

### DISCUSSION THREAD

**Discussion by MartinLA0416** [lencx/Noi#170] [Q&A]
<img width="2032" alt="截屏2024-05-21 15 09 22" src="https://github.com/lencx/Noi/assets/16625431/e386a935-5154-49c7-830c-919e2413c1bd">
The content that I input below does not show in the window of ChatGPT, but it shows in the windows of other AI.
我输入的内容不能显示在ChatGPT窗口里，但在其他AI窗口却可以显示

**Comment by kerzhaev** [comment]
I have same problem.

**Comment by ZhenhongDu** [comment]
You need to update the newest noi ask script. Openai has changed their website name. You can find more information in the issue part of Noi.

[a comment by an automated account is not shown]
